samples/Build2019Demo.Unity/Assets/Demo/Scripts/SampleDependencyVerifier.cs

```diff
@@ -0,0 +1,71 @@
+﻿#if UNITY_EDITOR
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Microsoft.MixedReality.SpectatorView.Samples
+{
+    [InitializeOnLoad]
+    public class SampleDependencyVerifier : MonoBehaviour
+    {
+        private const string FixSymlinksScript = "ResetSamples.ps1";
+        static SampleDependencyVerifier()
+        {
+            List<FileInfo> brokenLinks = new List<FileInfo>();
+            SearchForBrokenSymlink(new DirectoryInfo(Application.dataPath), brokenLinks);
+
+            if (brokenLinks.Count > 0)
+            {
+#if UNITY_EDITOR_WIN 
+                AttemptToFixOnWindows(brokenLinks);
+#else
+                EditorUtility.DisplayDialog("Broken Dependencies", $"Broken symbolic links detected, but automatic fix is not available for you development machine OS. Check GitHub for help, or file a new issue.\n\nBroken Symlinks:\n{string.Join("\n", brokenLinks)}", "Ok");
+#endif
+                Debug.LogError($"Broken Symlinks:\n{string.Join("\n", brokenLinks)}");
+            }
+        }
+
+        private static void AttemptToFixOnWindows(List<FileInfo> brokenLinks)
```

Comment:
Do we want users to automatically obtain dependencies with other licenses? IMO it may be worth keeping this an explicit step.

Regardless, can we target the 8/15 release with this change. This automated process seems like it may have some unintended consequences. I would like to avoid checking this into the stabilization branch and feel this should be verified via master, with our local development environments over the next few weeks.

Reply:
Removed file.